sdk/python/lib/pulumi/provider/experimental/server.py

```diff
@@ -108,18 +108,17 @@ async def GetPluginInfo(self, request, context) -> proto.PluginInfo:
 
     def create_grpc_invalid_properties_status(
         self, message: str, errors: Optional[List[InputPropertyErrorDetails]]
-    ):
-        status = grpc.Status()  # type: ignore[attr-defined]
+    ) -> grpc.Status:
+        status = grpc.Status()
         # We don't care about the exact status code here, since they are pretty web centric, and don't
         # necessarily make sense in this context.  Pick one that's close enough.
-        # type: ignore
```

Comment:
.value[0] no longer needed here but still needed below?

Reply:
Yeah, the first one is a `grpc.Status` that expects an StatusCode enum value according to the types, which is a tuple like `(0, "ok")`. Below is the protobuf type `status_pb2.Status` which takes an `int`.